### DISCUSSION TITLE
Make bank potion storage toggleable

### DISCUSSION THREAD

**Discussion by 4tet** [runelite/runelite#18493] [Ideas]
I bought this from mastering mixology, not realizing you can't disable it once you get it. Now it's taking up space on my bank tags sidebar and it's not a feature I see myself using, so I'd prefer to be able to hide it with a config setting/checkbox in RuneLite.

**Comment by sabiqalfajri** [comment]
response 2

**Comment by sabiqalfajri** [comment]
response 3
